ledfx/presets.py

```diff
@@ -438,7 +438,6 @@
                 "stretch_vertical": 67,
                 "test": False,
                 "resize_method": "Slow",
-                "half_beat": False,
```

Comment:
Why would half beatsetting be removed by a linter, even if currently the default value, that could change?

Reply:
ah, its in there twice!! the wrong one is removed however, we want the final value, so remove the True setting, let me check.

Reply:
confirmed, remove the True instance, not the false